_sass/components/_accordions.scss

```diff
@@ -76,6 +89,18 @@
 
 }
 
+.accordion-button,
```

Comment:
Selector `accordion-button_more` should be written in lowercase with hyphens<br>Selector `accordion-button_less` should be written in lowercase with hyphens